### DISCUSSION TITLE
Acknowledge ends with a 504 error

### DISCUSSION THREAD

**Discussion by mhgoff** [Security-Onion-Solutions/securityonion#4390] [Unsupported Versions]
I cannot seem to ack. any alarms.  Any time I try, I get an extremely long wait time, which finally ends with the following
![image](https://user-images.githubusercontent.com/85252479/120536631-f14a4d80-c3a9-11eb-8698-c8ff0554f385.png)

This has happened to me since I first installed the manager.  I have upgraded to the latest 2.3.51 with no change.

This is a distributed installation....separate manager, search and forward nodes.

Any thoughts?

**Comment by jertel** [comment]
More details for that 504 error should be available in `/opt/so/log/soc/sensoroni-server.log`.
